### PR TITLE
1.1.42 release notes

### PR DESCRIPTION
**Pull request by jameskoll-dbt**
## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

FYI @schottj @runleonarun

### CODE REVIEW

**Comment by jameskoll-dbt** [comment]
FYI @schottj @runleonarun 
I ran into trouble with the original SHA I was using so I had to do the release with a newer SHA and I updated the draft changelog text to reflect that. Wanted to give you a heads up in case you had already started reviewing the text before now.

File: website/docs/docs/dbt-cloud/dbt-cloud-changelog.md [hunk 1]
```diff
@@ -4,7 +4,20 @@ id: "cloud-changelog"
 sidebar_label: Changelog
 description: "Changelog for the dbt Cloud application"
 ---
+
+## dbt Cloud v1.1.42 (January 5th, 2022)
+
+#### New products and features
+
+We started the new year with a gift! Multi-tenant Team and Enterprise accounts can now use the new [Model timing](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-model-timing-tab) tab in dbt Cloud. You can use this to further explore long-running models to see if they need refactoring or rescheduling.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-changelog.md]:
```suggestion
We started the new year with a gift! Multi-tenant Team and Enterprise accounts can now use the new [Model timing](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-model-timing-tab) tab in dbt Cloud. You can use this tab to further explore long-running models to see if they need refactoring or rescheduling.
```

File: website/docs/docs/dbt-cloud/dbt-cloud-changelog.md [hunk 2]
```diff
@@ -4,7 +4,20 @@ id: "cloud-changelog"
...
+
+#### Performance improvements and enhancements
+
+* We added client-side naming validation for file or folder creation.
+* You can now see run duration for a completed run on the run details page.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-changelog.md]:
```suggestion
```

**Review by runleonarun — APPROVED** [review]
This is ready to merge, @jameskoll-dbt!

**Pull request merged**